example/PetStoreClient/src/Schema/Mapper/UserMapper.php

```diff
@@ -41,6 +45,9 @@ public function toSchema(array $payload): User
         if (isset($payload['userStatus'])) {
             $schema->setUserStatus($payload['userStatus']);
         }
+        if (empty($schema->toArray())) {
```

Comment:
looks a bit weird having this empty check in every mapper even if no oneOf is used